presto-docs/src/main/sphinx/prestissimo/prestissimo-features.rst

```diff
@@ -184,3 +184,184 @@ for cleanup. Only applicable when ``enable-old-task-cleanup`` is ``true``.
+
+``native_max_spill_file_size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``0``
+
+Specifies the maximum allowed spill file size, in bytes. If set to ``0``, there is no limit on the spill file size,
+allowing spill files to grow as large as necessary based on available disk space.
+This setting is crucial for managing disk space usage during operations that require spilling to disk.
+
+``native_max_spill_level``
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``4``
+
+Native Execution only. The maximum allowed spilling level for hash join build.
+0 is the initial spilling level, -1 means unlimited.
+
+``native_order_by_spill_memory_threshold``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Default value:** ``0``
+
+Native Execution only. Specifies the maximum memory, in bytes, that the `ORDER BY` operation can utilize before starting to spill data to disk.
+If set to ``0```, there is no limit on memory usage, potentially leading to large memory allocations for sorting operations.
+Configuring this threshold helps in managing memory usage more efficiently during `ORDER BY` operations.
+
+``native_row_number_spill_enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``true``
+
+Native Execution only. Enable row number spilling on native engine.
+
+``native_simplified_expression_evaluation_enabled``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``boolean``
+* **Default value:** ``false``
+
+Native Execution only. Enable simplified path in expression evaluation.
+
+``native_spill_compression_codec``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``varchar``
+* **Default value:** ``none``
+
+Native Execution only. Specifies the compression CODEC used to compress spilled data.
+Supported compression CODECs are: ZLIB, SNAPPY, LZO, ZSTD, LZ4, and GZIP.
+Setting this to ``none`` disables compression.
+
+``native_spill_file_create_config``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``varchar``
+* **Default value:** ``""``
+
+Native Execution only. Specifies the configuration parameters used to create spill files.
+These parameters are provided to the underlying file system, allowing for customizable spill file creation based on the requirements of the environment.
+The format and options of these parameters are determined by the capabilities of the underlying file system
+and may include settings such as file location, size limits, and file system-specific optimizations.
+
+``native_spill_write_buffer_size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``bigint``
+* **Default value:** ``1048576``
+
+Native Execution only. The maximum size in bytes to buffer the serialized spill data before writing to disk for IO efficiency.
+If set to zero, buffering is disabled.
```

Comment:
```suggestion
If set to ``0``, buffering is disabled.
```
Suggestion for consistency with similar statements earlier in the doc such as line 296.